### ISSUE TITLE
Modal body and footer render issues

### ISSUE THREAD

**Issue by KRRISH96** [Codebrahma/react-lite-ui#204]
**Describe the bug**

Modal `body` and `footer` aren't rendering when functions are passed.

**To Reproduce**

Send in the body and footer props as folloeing which returns null.
```js
<Modal
     body={
        () => <div>
             <img src="https://placeimg.com/300/300/nature" alt="nature" />
          </div>
      }
     footer={
        () => <Button type="success" borderless>Know more</Button>
     }
 />
```
**Expected behavior**

Should render `body` and `footer` when passed functions.

**Comment by rishichawda** [comment]
Add check for `typeof footer === 'function'` and execute. Should be good to go. 👍

**Comment by KRRISH96** [comment]
Fixed in #200 Commit [96f7ea2](https://github.com/Codebrahma/react-lite-ui/commit/96f7ea2555bca612ade267780f20c27daddc1a55)
